README.md

```diff
@@ -79,6 +79,24 @@ do replacements. This simplifies the writing of the code into a
 constructor that registers preprocessor callbacks or ast matchers and
 a postprocessing phase.
 
+### `clangmetatool::propagation::ConstantCStringPropagator`
+
+This provides infrastructure (utilizing
+`clangmetatool::propagation::ConstantPropagator` and
+`clangmetatool::propagation::PropagationVisitor`) to propagate constant
+C-style string values over the program. Resulting in the true value of a
+variable wherever the value is deterministic and "<UNRESOLVED>" anywhere else.
+
+#### `clangmetatool::propagation::ConstantPropagator` and `clangmetatool::propagation::PropagationVisitor`
+
+These two classes provide the boilerplate to create infrastructure
+to propagate constants of arbitrary types through the control flow graph
+of the program in such a way that anywhere the constant value of a variable
+would be deterministic one may query its value at that point.
+
+These classes are private to the library, but additional propagators could be easily
+made using these facilities.
+
```

Comment:
Those should move into the "Constant Propagation" section as well

Reply:
Done